{{cookiecutter.project_name}}/monitoring/model_monitoring_setup_template.py

```diff
@@ -0,0 +1,278 @@
+{%raw%}
+def get_notebook(timestamp_col, model_version_col, prediction_col, problem_type, table_name, granularity, model_name):
+    return f'''# Databricks notebook source
+# Install data monitoring client library. Eventually this will be included in MLR
+%pip install "https://databricks-mvn.s3.amazonaws.com/databricks-data-monitoring/databricks-data-monitoring-buZrBwyN.tar.gz?AWSAccessKeyId=AKIAJLF6DMKWZKIYMPAQ&Expires=1990213870&Signature=JouwkAylFhuviaVDArkznIZQkgM%3D"
+
+# COMMAND ----------
+
+# import libraries
+from pyspark.sql import types as T
+import databricks.data_monitoring as dm
+from databricks.data_monitoring.analysis import analysis_type
```

Comment:
We usually do not want users to import like this. Those classes can be directly accessed like `dm.analysis.InferenceLog`.
```suggestion
```